src/components/molecules/MenuItems/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Button from '../../atoms/Button/index';
+
+const MenuItems = [
```

Comment:
get menu items from structure.json

Reply:
Created file data.json for storing the data